### DISCUSSION TITLE
First day of the week in widget

### DISCUSSION THREAD

**Discussion by ownmas** [community/community#3468] [Mobile] [answered]
Please make Monday as a first day of the week for Europeans. Apple has an [easy API](https://developer.apple.com/documentation/foundation/nscalendar/1408310-firstweekday) for this
![image](https://user-images.githubusercontent.com/10982852/109883713-dd992900-7c8c-11eb-8ebc-697523c604d9.jpeg)
![image](https://user-images.githubusercontent.com/10982852/109884010-55ffea00-7c8d-11eb-9429-590238a64d73.png)

**Comment by PranamBhat** [comment]
@ownmas It will be a new feature request. Can you please mention the same over feedback. Meanwhile you can check here : [https://github.community/] (https://github.community/)

Kindly mark this as "**Answered**" and close/leave the discussion. Thank you!

**Comment by candyho** [comment, marked as answer]
Hey thanks for reaching out again! I've replied to another discussion for consolidation purpose - https://github.com/github/feedback/discussions/3374
